### ISSUE TITLE
OpenGL support should be available on OS X

### ISSUE THREAD

**Issue by darkaegisagain** [PixarAnimationStudios/OpenSubdiv#920]
I was able to build the OpenSubdiv source with OpenGL transform feedback support under OS X, although OS X does not have a complete implementation of OpenGL 4.2 which is required by the current build it has enough features to make this work correctly.

This provides hardware acceleration of OpenGL on OS X and was a simple change in the cmake build system.

You can add me as a contributor and I can make the required changes.

**Comment by jtran56** [comment]
Filed as internal issue #142046.

**Comment by darkaegisagain** [comment]
FYI.. I currently work for nVidia and I worked on the OpenGL / OpenCL Framework at Apple for 10 years. Feel free to ask for help, I may know a few things.. 👍

**Comment by significant-bit** [comment]
+1 interest in maintaining Mac support

**Comment by davidgyu** [comment]
Yes, indeed! The OpenSubdiv XFB implementation should work on Mac, and it does when OpenSubdiv is using GLEW but GLEW should not be required. We'd welcome a PR to fix to the CMake build. Thanks!
